macros/helpers/prime_line.cfg

```diff
@@ -30,6 +30,19 @@ gcode:
     {% set prime_line_y = params.START_Y|default(prime_line_y)|float %}
     {% set prime_line_direction = params.LINE_DIRECTION|default(printer["gcode_macro _USER_VARIABLES"].prime_line_direction)|string|upper %}
 
+    # Redefine start position from fl_size 
+    {% set fl_size=printer['gcode_macro START_PRINT'].fl_size %}
+    {% if fl_size != "0_0_0_0" %}
+        {% set prime_line_margin = params.LINE_MARGIN|default(printer["gcode_macro _USER_VARIABLES"].prime_line_margin|float) %}
```

Comment:
This will break klippain on moonraker update if a fixed value default isn't added.
We had that topic on mmu already.